db/seeds.rb

```diff
@@ -6,23 +6,32 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
-Item.create(title: "Unicorn Pie",
-  description: "Gooooood",
-  price: 8,
-  status: "active")
-Item.create(title: "Medusa Hair Pasta",
-  description: "Gooooood",
-  price: 8,
-  status: "active")
-Item.create(title: "Pixie Bread Stix",
-  description: "Gooooood",
-  price: 8,
-  status: "active")
-Item.create(title: "Phoenix Spicy Curry",
-  description: "Gooooood",
-  price: 8,
-  status: "active")
-Item.create(title: "Titan Burger",
-  description: "Gooooood",
-  price: 8,
-  status: "active")
+unicorn_pie       = Item.create(title: "Unicorn Pie",
+                                description: "Gooooood",
+                                price: 8,
+                                status: "active")
+medusa_hair_pasta = Item.create(title: "Medusa Hair Pasta",
+                                description: "Gooooood",
+                                price: 8,
+                                status: "active")
+pixie_bread_stix  = Item.create(title: "Pixie Bread Stix",
+                                description: "Gooooood",
+                                price: 8,
+                                status: "active")
+phoenix_curry     = Item.create(title: "Phoenix Spicy Curry",
```

Comment:
Useless assignment to variable - `phoenix_curry`.<br>Unnecessary spacing detected.